packages/stark-ui/src/modules/input-mask-directives/directives/timestamp-mask.directive.ts

```diff
@@ -55,7 +55,7 @@ export class StarkTimestampMaskDirective extends MaskedInputDirective implements
 	 */
 	/* tslint:disable:no-input-rename */
 	@Input("starkTimestampMask")
-	public maskConfig: StarkTimestampMaskConfig;
+	public maskConfig: StarkTimestampMaskConfig = { format: "DD-MM-YYYY hh:mm:ss" };
```

Comment:
Can you create a constant for the format string so we can re-use it here and also in the `private readonly defaultTimestampMaskConfig` property? Because in fact we are now using a different format here than the one over there :(

Reply:
Ah yes indeed. 🤔 I changed it to this.
```
/**
	 * Default configuration.
	 * It will be merged with the configuration passed to the directive.
	 */
	private readonly defaultTimestampMaskConfig: StarkTimestampMaskConfig = {
		format: DEFAULT_DATE_TIME_FORMAT
	};

	/**
	 * Configuration object for the mask to be displayed in the input field.
	 */
	/* tslint:disable:no-input-rename */
	@Input("starkTimestampMask")
	public maskConfig: StarkTimestampMaskConfig = this.defaultTimestampMaskConfig;
```